content/scalability/docs/control-plane.md

```diff
@@ -0,0 +1,164 @@
+
+Scaling large applications requires infrastructure to adapt to become fully ready (e.g. warming load balancers). To control the speed of scaling make sure you are scaling based on the right metrics for your application. CPU and memory scaling may not accurately predict your application constraints and using custom metrics (e.g. requests per second) in Kubernetes Horizontal Pod Autoscaler](hPA) may be a more accurate scaling option.
+
+To use a custom metric see the examples in the [Kubernetes documentation](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics).
+
+## Scale nodes and pods down safely using Pod Disruption Budgets or Karpenter TTL Settings
+
+Use Karpenter’s [time to live (TTL)](https://aws.github.io/aws-eks-best-practices/karpenter/#use-timers-ttl-to-automatically-delete-nodes-from-the-cluster) settings to replace instances after they’ve been running for a specified amount of time. Self managed node groups can use the `max-instance-lifetime` setting to cycle nodes automatically. Managed node groups do not currently have this feature but you can track the request [here on GitHub](https://github.com/aws/containers-roadmap/issues/1190).
+
+Remove nodes when they have no running workloads using scale down settings in the Kubernetes Cluster Autoscaler and Karpenter. Set Pod Disruption budgets (PDB) on your workloads to avoid application outages when nodes are removed. Consider enabling [consolidation](https://aws.github.io/aws-eks-best-practices/karpenter/#configure-requestslimits-for-all-non-cpu-resources-when-using-consolidation) in your Karpenter provisioner to replace nodes that are not fully utilized.
+
+You can set the node scale down threshold in the Kubernetes Cluster Autoscaler with the [`--scale-down-utilization-threshold`](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#how-does-scale-down-work) or in Karpenter you can use the ttlSecondsAfterEmpty provisioner setting.
+
+## Use Client-Side Cache when running Kubectl
+
+Using the kubectl command, you can add additional load to the Kubernetes control plane. You should avoid running scripts or automation that uses kubectl repeatedly (e.g. in a for loop) because each command makes dozens or hundreds of calls to the API server.
+
+This can be especially impactful if you run kubectl from a container without a client-side cache. The client-side cache is a local file that caches information from the cluster about APIs available and Custom Resources (CR). Refreshing the cache happens by default every 10 minutes when kubectl is run and will make hundreds of API calls. You’re likely to run into API throttling issues if you run kubectl commands frequently without a local cache.
```

Comment:
Is there a flag or example we can provide on how client side caching can be enabled?

Reply:
Client side caching is always enabled by default and currently there is no way to disable it.

Reply:
Got it. If it is turned on by default (and cannot be disabled), then is this an actionable recommendation? Or is it more of an info for the reader? I agree there is value in leaving it here, but maybe just clarifying that this enabled by default (or something to that extend) will help the reader